database/insertData/03_template_A_featureShowcase.js

```diff
@@ -1102,6 +1102,161 @@ exports.queries = [
+                      index: 9
+                      title: "Search ATC Codes"
+                      elementTypePluginCode: "search"
+                      category: QUESTION
+                      isRequired: false
+                      parameters: {
+                        label: "Lookup ATC codes"
+                        description: "Start typing a drug name"
+                        placeholder: "Search medicines"
+                        icon: "pills"
+                        minCharacters: 3
+                        multiSelect: true
+                        source: {
+                          operator: "graphQL",
+                          children: [
+                            "query GetEntitiesByName($search: String!) {entities (filter: { code: \\"\\" description: $search type: \\"[drug]\\" match: \\"contains\\"},offset: 0,first: 25) {data {        code,     description,        type,        uid  }, totalLength }}",
+                            "https://codes.msupply.foundation:2048/graphql",
+                            [
+                              "search"
+                            ],
+                            {
+                              operator: "objectProperties",
+                              children: [
+                                "search.text"
+                              ]
+                            },
+                            "entities.data"
+                          ]
+                        }
+                        displayFormat: {
+                          title: "\${description}",
+                          description: "ATC code: \${code}"
+                        }
+                        resultFormat: {
+                          title: "\${description}",
+                          description: "\${code}"
+                        }
+                      }
+                    }
+                    {
+                      code: "searchOrg"
+                      index: 11
+                      title: "Search Demo (organisations)"
+                      elementTypePluginCode: "search"
+                      category: QUESTION
+                      isRequired: false
+                      helpText: "This one has no \`displayFormat\` prop, so it falls back to a generic \\"default\\" display"
+                      parameters: {
+                        label: "Search for an organisation"
+                        description: "Please enter the organisation's registration code (min 6 chars)"
+                        placeholder: "Search organisations"
+                        icon: "building"
+                        minCharacters: 6
+                        source: {
+                          operator: "graphQL",
+                          children: [
+                            "query GetOrgs($registration: String!) { organisations(filter: {registration: {startsWithInsensitive: $registration}}) { nodes { name registration } } }",
+                            "",
```

Comment:
Also maybe better to use `graphqlEndpoint` here so it's more descriptive

Reply:
Yup, done with [this one](https://github.com/openmsupply/application-manager-server/pull/380#discussion_r642700688)